### PR TITLE
fix pandas support

### PR DESCRIPTION
**Pull request by jeppe742**
pandas 2.2 doesn't support SQLAlchemy <2.0 anymore

https://github.com/pandas-dev/pandas/issues/57049

### CODE REVIEW

**Comment by jeppe742** [comment]
create_package

File: pyproject.toml [hunk 1]
```diff
@@ -46,7 +46,7 @@ hvac = { version = "~0.11.2", optional = true }
 
 fsspec = { version = "2022.8.2" }
 
-trino = { version = "0.319.0", optional = true, extras = ['sqlalchemy'] }
+trino = { version = "0.327.0", optional = true, extras = ['sqlalchemy'] }
```
Comment (george-zubrienko) [inline pyproject.toml:49]:
```suggestion
trino = { version = "^0.327.0", optional = true, extras = ['sqlalchemy'] }
```

**Review by george-zubrienko — APPROVED** [review]
LGTM, but please bump minor for this as it breaks pandas <2.2 clients

File: pyproject.toml (hunk 1 above)
Comment (george-zubrienko) [reply pyproject.toml:49]: remember to run `poetry update` after this

**Pull request merged**